app/services/hooks/useLocTrackingStatus.js

```diff
@@ -0,0 +1,54 @@
+import BackgroundGeolocation from '@mauron85/react-native-background-geolocation';
+import { useEffect, useState } from 'react';
+
+import LocationService from '../LocationService';
+
+/**
+ * Hook used to check the location tracking status of a user
+ * @param {{canTrack: boolean, reason: Reason, isRunning: boolean, hasPotentialExposure: boolean}} status
+ */
+export const useLocTrackingStatus = () => {
+  const initialState = {
+    canTrack: undefined,
+    reason: undefined,
+    isRunning: undefined,
+    hasPotentialExposure: undefined,
```

Comment:
This probably shouldn't be here anymore... but.. can remove in a different PR

Reply:
I'm fine to just remove now. The only reason I included it is because it currently matches the shape of the object returned from `LocationService.checkStatus()`, so I thought it was clearer.

But if we just remove it here there isn't really a downside, so if we plan to remove it from `checkStatus()`, I'll just remove this.